appvm-scripts/usrbin/qubes-session

```diff
@@ -27,6 +27,14 @@
 
 loginctl activate "$XDG_SESSION_ID"
 
+# Now import the environment from the systemd user session.
+# This is necessary to enable users to configure their
+# Qubes environment using the standard environment.d
+# facility.  Documentation for the facility is at:
+# https://www.freedesktop.org/software/systemd/man/environment.d.html
+eval `systemctl --user show-environment | sed 's/^/export /'`
```

Comment:
Please use `$(...)` instead of the deprecated syntax.

Reply:
Done.